### DISCUSSION TITLE
File name issue

### DISCUSSION THREAD

**Discussion by keltt929** [gildas-lormeau/SingleFile#1427] [General]
After updading to 1.22.46 (Chrome), the file name become very strange in every web page I visited.
For example: 80c89dc8-8de3-43bb-921a-c1b19d1aee50.htm

It works well in 1.22.45 (Firefox). Not sure if this issue is related to Chrome or Snglefile. But the file name is no problem when I try to save file by using the "Save File" in Chrome. Please have a look for this issue. Thank you.

**Comment by gildas-lormeau** [comment]
This is a known issue, see https://github.com/gildas-lormeau/SingleFile?tab=readme-ov-file#known-issues
